### PR TITLE
📖 Update triggers for feature tests

### PR DESCRIPTION
**Pull request by smoshiur1237**
This will update the feature tests triggers on the PR to run them independently.

### CODE REVIEW

**Comment by smoshiur1237** [comment]
/override metal3-centos-e2e-integration-test-main metal3-ubuntu-e2e-integration-test-main 

Tests are not relevant. Change in docs only.

**Comment by smoshiur1237** [comment]
/override metal3-centos-e2e-integration-test-main metal3-ubuntu-e2e-integration-test-main

**Comment by smoshiur1237** [comment]
/override metal3-centos-e2e-integration-test-main metal3-ubuntu-e2e-integration-test-main

**Review by Rozzii — COMMENTED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -5,7 +5,9 @@
...
 [![Ubuntu E2E Integration main build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3-periodic-ubuntu-e2e-integration-test-main&subject=Ubuntu%20e2e%20integration%20main)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3-periodic-ubuntu-e2e-integration-test-main/)
 [![CentOS E2E Integration main build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3-periodic-centos-e2e-integration-test-main&subject=Centos%20e2e%20integration%20main)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3-periodic-centos-e2e-integration-test-main/)
-[![CentOS E2E feature main build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3-periodic-centos-e2e-feature-test-main/&subject=CentOS%20E2E%20feature%20main)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3-periodic-centos-e2e-feature-test-main/)
+[![CentOS E2E feature main pivot build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3-periodic-centos-e2e-feature-test-main-pivoting/&subject=CentOS%20E2E%20feature%20main%20pivot)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3-periodic-centos-e2e-feature-test-main-pivoting/)
```
Comment (Sunnatillo) [inline README.md:8]: `pivot` or `pivoting`?

**Pull request merged**